### ISSUE TITLE
question levels

### ISSUE THREAD

**Issue by shanaka75** [ua2004/Millionaire-3D#37]
is it possible to set
15 different question sets ? lvl 1-15
so game randomly picks a question from corresponding level ?

ex: your in question number 5 so game will pick a random question from lvl 5 question set.

this will make game more live and challangeing and similar to real game.

**Comment by ua2004** [comment]
Yes, it is possible. However, this will make things harder for people who create questions. How will they classify the question difficulty level? They will argue and hesitate whether this question is more suitable for level 7, 8 or 9 as their difficulty is almost similar. In most implementations of this game there are only three difficulty levels:

1. Questions 1-5
2. Questions 6-10
3. Questions 11-15

But we will also add a separate level 4 for the last 15th question since it is worth a million and definitely should be harder than question 11 or 12.
